clients/admin-ui/src/types/api/models/DataCategory.ts

```diff
@@ -28,4 +28,8 @@ export type DataCategory = {
    * Denotes whether the resource is part of the default taxonomy or not.
    */
   is_default?: boolean;
+  /**
```

Comment:
we'll want to make sure that this is generated from the `npm run openapi:generate` command. you'd have to run it against fidesplus (followed by an `npm run format` to apply prettier to all the new files)